Userland/Libraries/LibCore/Command.h

```diff
@@ -23,4 +28,38 @@ struct CommandResult {
 ErrorOr<CommandResult> command(DeprecatedString const& program, Vector<DeprecatedString> const& arguments, Optional<LexicalPath> chdir);
 ErrorOr<CommandResult> command(DeprecatedString const& command_string, Optional<LexicalPath> chdir);
 
+class Command {
+public:
+    struct ProcessOutputs {
+        ByteBuffer standard_output;
+        ByteBuffer standard_error;
+    };
+
+    static ErrorOr<OwnPtr<Command>> create(StringView command, char const* const arguments[]);
+
+    Command(pid_t pid, NonnullOwnPtr<Core::File> stdin_file, NonnullOwnPtr<Core::File> stdout_file, NonnullOwnPtr<Core::File> stderr_file);
+
+    ErrorOr<void> write(StringView input);
+
+    ErrorOr<void> write_lines(Span<DeprecatedString> lines);
```

Comment:
I think this can take a Span of StringViews instead, since it completes synchronously.